socs/agents/vantagepro2/drivers.py

```diff
@@ -241,27 +241,27 @@ def receive_data(self):
         loop_data['packet_type'] = byte_data[4]
         loop_data['next_record'] = byte_data[5]
         loop_data['barometer'] = byte_data[6] / 1000.0
-        loop_data['temp_inside'] = byte_data[7] / 10.0
+        loop_data['temp_inside'] = (byte_data[7] / 10.0 - 32) * (5 / 9)
```

Comment:
Please write a function that converts F to C, and use it on all of these.  This is like the classic example of when should use a function.

Reply:
you're right :(